stats/lib/handler.js

```diff
@@ -126,20 +126,14 @@ const errorHandler = (res, err, logger) => {
     res.end(err.message)
   } else {
     logger.error(err)
-    res.statusCode = 500
-    res.end('Internal Server Error')
+    status(res, 500)
   }
 
   if (res.statusCode >= 500) {
     Sentry.captureException(err)
   }
 }
 
-const notFound = (res) => {
-  res.statusCode = 404
-  res.end('Not Found')
-}
```

Comment:
Before, we set the response body to 'Not Found'.

What happens after the upgrade, when we call `status(res, 404)`? Will the status function set the response body to the status message corresponding with the status code?

Reply:
Yes! This was my motivation for introducing this method :)

https://github.com/juliangruber/http-responders/blob/5820864da9cb4728d4d86e76184eb554a92c16ed/index.js#L5-L8